src/include/type/type_util.h

```diff
@@ -36,12 +36,14 @@ class TypeUtil {
       case TypeId::PARAMETER_OFFSET:
         return 4;
       case TypeId::BIGINT:
-      case TypeId::DECIMAL:
+      case TypeId::REAL:
       case TypeId::TIMESTAMP:
         return 8;
       case TypeId::VARCHAR:
       case TypeId::VARBINARY:
         return storage::VARLEN_COLUMN;
+      case TypeId::DECIMAL:
```

Comment:
Just doing this probably might be wrong. You need to change the constructor of the SQL table too. I have already done that in my PR, so that might work.

Reply:
I think we punt that type of change until the end-to-end DECIMAL support PR, especially if you've made those changes already. I wouldn't be able to exercise them yet.